cmd/serve.go

```diff
@@ -13,5 +13,6 @@ func serveCommand(logger log.Logger, appConfig *config.Shield) *cli.Command {
 	}
 	c.AddCommand(proxyCommand(logger, appConfig))
 	c.AddCommand(adminCommand())
```

Comment:
@krtkvrm can you remove this admin command now? I added this for admin APIs only. Also, we need to have a single command that starts proxy as well as admin APIs.